libs/island-ui/core/src/lib/ActionCard/ActionCard.tsx

```diff
@@ -61,20 +67,31 @@ const defaultUnavailable = {
 } as const
 
 export const ActionCard: React.FC<ActionCardProps> = ({
+  date,
   heading,
   text,
   cta: _cta,
   tag: _tag,
   unavailable: _unavailable,
   progressMeter: _progressMeter,
 }) => {
-  const cta = { ...defaultCta, ..._cta }
+  const cta = {
+    ...defaultCta,
+    ..._cta,
+    variant:
+      _cta.variant === 'secondary'
+        ? ('text' as const)
+        : _cta.variant === 'ghost'
+        ? ('ghost' as const)
+        : ('primary' as const),
```

Comment:
We should probably change this **later** on to represent the button variants, that will probably cause less confusion and we would not have to explain what each variant will render